homeassistant/components/cover/isy994.py

```diff
@@ -0,0 +1,115 @@
+"""
+Support for ISY994 covers.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/cover.isy994/
+"""
+import logging
+from typing import Callable  # noqa
+
+from homeassistant.components.cover import CoverDevice, DOMAIN
+from homeassistant.components.isy994 import (ISYDevice, NODES, PROGRAMS, ISY,
```

Comment:
You import `ISY`, a pointer to the instantiated client. 

If it was not initialized by the time you imported it here, this `ISY` would have been None

Today bootstrap calls the component's `setup` before importing platforms, ensuring that the global `ISY` in the component is initialized, but this is still not the recommended way to do it.